modules/programs/java.nix

```diff
@@ -37,10 +37,7 @@ in {
     home.packages = [ cfg.package ];
 
     home.sessionVariables = {
-      JAVA_HOME = fileContents (pkgs.runCommandLocal "java-home" { } ''
-        source "${cfg.package}/nix-support/setup-hook"
-        echo "$JAVA_HOME" > $out
-      '');
+      JAVA_HOME = cfg.packages.home;
```

Comment:
```suggestion
      JAVA_HOME = cfg.package.home;
```

Reply:
Ooops, damn mobile :D

Should be fixed via 2350f5a